src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+  notification.style.marginTop = '10px';
+
+  notification.innerHTML = `<h2 class="title">${title}</h2>
+  <p>${description}</p>`;
+
+  const lastNotification = document.querySelector('.notification:last-of-type');
```

Comment:
it's not necessary to save `lastNotification`. just set the `posTop` and `posRight` from argumenta